jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -278,7 +278,7 @@ exprt::operandst java_string_library_preprocesst::process_parameters(
   exprt::operandst ops;
   for(const auto &p : params)
   {
-    symbol_exprt sym(p.get_identifier(), p.type());
+    const symbol_exprt sym(p.get_identifier(), p.type());
     ops.push_back(sym);
```

Comment:
I'd go the opposite direction: not use `const` and use `ops.push_back(std::move(sym));` or even just do `ops.emplace_back(symbol_exprt(p.get_identifier(), p.type()));` - `sym` doesn't add much value anyway.